### PR TITLE
Test&Score,Predictions: output predictions and probabilities

### PR DESCRIPTION
**Pull request by nikicc**
Test and score now also returns predicted values and probabilities

### CODE REVIEW

File: Orange/evaluation/testing.py [hunk 1]
```diff
@@ -202,6 +202,53 @@ def get_fold(self, fold):
 
         return results
 
+    def get_table_with_probabilities(self, model_names):
+        """
+            Return the Orange.data.Table with data, predictions and probabilities (if the task is classification).
```
Comment (kernc) [inline Orange/evaluation/testing.py]: The standard is for docstrings to be aligned with opening tri-quotes (cf. e.g. `np.hstack()`).

File: Orange/evaluation/testing.py [hunk 2]
```diff
@@ -202,6 +202,53 @@ def get_fold(self, fold):
...
+
+            :param model_names: A list of strings containing learners' names.
```
Comment (kernc) [inline Orange/evaluation/testing.py]: Would you consider writing any new docstrings in [Napoleon style](http://sphinxcontrib-napoleon.readthedocs.org/en/latest/index.html)?

Comment (janezd) [inline Orange/evaluation/testing.py]: +1 for Napoleon. Much easier to read.

Comment (nikicc) [inline Orange/evaluation/testing.py]: Sure. Did we agree on whether to use Google or NumPy version or can I choose?

**Pull request merged**